src/common/frontend/helpers/page.js

```diff
@@ -1,24 +1,33 @@
 import React from 'react';
 import { render } from 'react-dom';
-import { createStore, applyMiddleware } from 'redux';
-import thunk from 'redux-thunk';
-import createLogger from 'redux-logger';
-import { Provider } from 'react-redux';
-import reducer from '../reducers';
 import nconf from 'nconf';
+import urljoin from 'url-join';
 
-const middleware = nconf.get('NODE_ENV') !== 'production' ? applyMiddleware(thunk, createLogger()) : applyMiddleware(thunk);
+import { 
+  ApolloClient, 
+  ApolloProvider,
+  createNetworkInterface,
+} from 'react-apollo';
 
 export function mount(Page, func) {
   if (typeof document !== 'undefined') {
-
-    const state = window.__STATE__;
-    const store = createStore(reducer, state, middleware);
-
+    
+    const networkInterface = createNetworkInterface({
```

Comment:
The network interface is basically how the "client" connects to the server to load data. For the real client, this is simply just the POST route `/graphql`. For the server when we do server rendering, this is a specialized "local" network interface so that the server can more efficiently query itself to load data. One of the benefits here of using Apollo and GraphQL is that all the code we write is the same. It is "isomorphic", and then the different network interface handles how to fetch the data depending on if you're client or server.

We previously did hacky things like having an "adapter" which defaulted to a clientAdapter [here](https://github.com/PennChristianJournal/LocustWalk/blob/v2/src/common/frontend/actions/articles.js#L31) which we overwrote with a specialized server adapter for server rendering.

This was annoying though because we had to manually handle the preloading stuff here
https://github.com/PennChristianJournal/LocustWalk/blob/v2/src/common/controllers/articles.js#L22